GenericsHomework/Node.cs

```diff
@@ -0,0 +1,107 @@
+﻿//using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace GenericsHomework;
+
+public class Node<T>
+{
+    private T _value { get; set; }
+    public Node<T>? next { get; set; }

```

Comment:
next setter should be private
```suggestion
    public Node<T> next { get; private set; }

```

Reply:
Fixed that, thanks for catching that!